backend/core/src/listings/listings.module.ts

```diff
@@ -50,7 +57,7 @@ if (process.env.REDIS_USE_TLS !== "0") {
   controllers: [ListingsController],
 })
 // We have to manually disconnect from redis on app close
-export class ListingsModule implements OnModuleDestroy {
+export class ListingsModule implements OnModuleDestroy, OnModuleInit {
```

Comment:
@pbn4 , I think you mentioned this in stand-up, can you explain the addition of `OnModuleInit` here and the purpose of resetting the cache before quitting redis?